docs/notebooks/figanos_docs.ipynb

```diff
@@ -54,12 +54,32 @@
   },
```

Comment:
<p>I think it would be a better example with the ouranos logo. I don't think people will understand it is available if we don't show it here.</p><p> </p><p>I would also show how to install the Ouranos logo in this notebook.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/figanos/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>